### PR TITLE
Release `0.28.0-rc.0`

### PR DESCRIPTION
**Pull request by neysofu**
This is the first release candidate for `v0.28.0`. `NEWS.md` has been updated with the release notes from [this](https://github.com/graphprotocol/graph-node/compare/v0.27.0...1435fceb5840c479209c54330ce0dc6b5a3c1b6e) commit range.

### CODE REVIEW

**Comment by neysofu** [comment]
On hold; waiting for https://github.com/graphprotocol/graph-node/issues/3936.

**Comment by neysofu** [comment]
Ready for review. Will cherrypick https://github.com/graphprotocol/graph-node/commit/6387465040b98f926880f069c655b8f5ea1f1b4b on top of this branch after merging with `master`.

File: NEWS.md [hunk 1]
```diff
@@ -2,13 +2,103 @@
...
+- Index nodes now have an endpoint to perform block number to canonical hash conversion, which will unblock further work towards multichain support. [#3942](https://github.com/graphprotocol/graph-node/pull/3942)
+- `_meta.block.timestamp` is now available for all EVM chains. [#3738](https://github.com/graphprotocol/graph-node/pull/3738), [#3902](https://github.com/graphprotocol/graph-node/pull/3902)
+- The `deployment_eth_rpc_request_duration` metric now also observes `eth_getTransactionReceipt` requests' duration. [#3903](https://github.com/graphprotocol/graph-node/pull/3903)
+- New Prometheus metrics `query_parsing_time` and `query_validation_time` for monitoring query processing performance. [#3760](https://github.com/graphprotocol/graph-node/pull/3760)
+- New command `graphman config provider`, which shows what providers are available for new deployments on a given network and node. [#3816](https://github.com/graphprotocol/graph-node/pull/3816)
+  E.g. `$ graphman --node-id index_node_0 --config graph-node.toml config provider mainnet`
+- Experimental support for GraphQL API versionining has landed. [#3185](https://github.com/graphprotocol/graph-node/pull/3185)
```
Comment (azf20) [inline NEWS.md:26]: Versioning

File: NEWS.md [hunk 2]
```diff
@@ -2,13 +2,103 @@
...
+- Node IDs can now contain any character. The Docker start script still replaces hyphens with underscores for backwards compatibility reasons, but this behavior can be changed with the `GRAPH_NODE_ID_USE_LITERAL_VALUE` environment variable. With this new option, you can now seamlessly use the K8s-provided host names as node IDs, provided you reassign your deployments accordingly. [#3688](https://github.com/graphprotocol/graph-node/pull/3688)
+- You can now use the `conn_pool_size` option in TOML configuration files to configure the connection pool size for Firehose providers. [#3833](https://github.com/graphprotocol/graph-node/pull/3833)
+- Index nodes now have an endpoint to perform block number to canonical hash conversion, which will unblock further work towards multichain support. [#3942](https://github.com/graphprotocol/graph-node/pull/3942)
+- `_meta.block.timestamp` is now available for all EVM chains. [#3738](https://github.com/graphprotocol/graph-node/pull/3738), [#3902](https://github.com/graphprotocol/graph-node/pull/3902)
```
Comment (azf20) [inline NEWS.md:21]: "available for subgraphs indexing EVM chains"

File: NEWS.md [hunk 3]
```diff
@@ -2,13 +2,103 @@
...
+- Fixed a crash caused by bad block data from the provider. [#3944](https://github.com/graphprotocol/graph-node/pull/3944)
+- Fixed some minor Firehose connectivity issues via TCP keepalive, connection and request timeouts, and connection window size tweaks. [#3822](https://github.com/graphprotocol/graph-node/pull/3822), [#3855](https://github.com/graphprotocol/graph-node/pull/3855), [#3877](https://github.com/graphprotocol/graph-node/pull/3877), [#3810](https://github.com/graphprotocol/graph-node/pull/3810), [#3818](https://github.com/graphprotocol/graph-node/pull/3818)
+- Copying private data sources' tables across shards now works as expected. [#3836](https://github.com/graphprotocol/graph-node/pull/3836)
+
+### Performance improvements
+
+- Firehose GRPC stream requests are now compressed with `gzip`, if the server supports it. [#3893](https://github.com/graphprotocol/graph-node/pull/3893)
+- Memory efficiency improvements within the entity cache. [#3594](https://github.com/graphprotocol/graph-node/pull/3594)
+- Identical queries now benefit from GraphQL validation caching, and responses are served faster. [#3759](https://github.com/graphprotocol/graph-node/pull/3759)
```
Comment (azf20) [inline NEWS.md:47]: @lutter did we not revert this change?

**Review by azf20 — APPROVED** [review]
I think this looks good! Some small comments

File: NEWS.md (hunk 1 above)
Comment (neysofu) [reply NEWS.md]: Fixed! Thanks for the catch

File: NEWS.md (hunk 2 above)
Comment (neysofu) [reply NEWS.md]: 👍

**Pull request merged**